src/models/converters/transactions/tests/transfer_type_checks.rs

```diff
@@ -1,213 +0,0 @@
-use crate::models::backend::transactions::{MultisigTransaction, Confirmation};
```

Comment:
Do we have similar tests for the new logic (with the same input)?

Reply:
Not yet 😓  working on it

Reply:
Just restored the methods and adjusted them to the new methods like we talked